### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Daniel-Warner-X**
Update readme to reflect Navie release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,97 @@
...
+1. **Install [the AppMap Plugin](https://plugins.jetbrains.com/plugin/16701-appmap)** from within the code editor or from the marketplace.  
+
+2. **Sign in with an email address, or with GitHub or GitLab** and Navie will be available in `@explain` mode. This enables Navie to respond to general coding and development questions and answer questions about using AppMap data.
+
+3. **Ask Navie** for guidance recording AppMap data specific to interactions or code scenarios you're interested in analyzing.
+
+   
+## Examples
+Here are some examples of Navie making context-aware suggestions, providing tactical solutions, and reasoning about the larger context of the specific code being worked on.
+
```
Comment (kgilpin) [inline README.md:26]: Looks good overall, we just have to find a way to get an IntelliJ example in here - that's our biggest segment.

**Review by kgilpin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Daniel-Warner-X) [reply README.md:26]: Java example added.

Comment (jansorg) [reply README.md:26]: If you wanted to change the description, which appears on the JetBrains Marketplace, then https://github.com/getappmap/appmap-intellij-plugin/blob/develop/description.md would be the file to change.

Comment (Daniel-Warner-X) [reply README.md:26]: Thanks @jansorg I updated `description.md`

File: description.md [hunk 2]
```diff
@@ -1,82 +1,104 @@
...
-Code traces and visualizations from AppMap show how your code really works, including APIs, database queries, web service request flows, security architecture, and more.
+* Navie - AI code architect.
+* AppMap data - Runtime traces of your application runtime.
+* AppMap diagrams - Interactive visualizations of AppMap data.
 
-AppMap’s AI coding assistant, Navie, will use data and visualizations to co-develop new features, fix complex bugs, optimize code performance and keep your code secure.
+By using AppMap data, Navie is the first AI code architect with the context to understand how your app works when it runs, like a principal engineer or software architect. AppMap data provides Navie with accurate information about application behavior, APIs, database queries, and more, providing valuable context so you can ask more challenging questions and get better answers.
 
-AppMap and Navie will help you:
+![image](https://github.com/getappmap/appmap-intellij-plugin/assets/1229326/ed36497e-d21d-4b79-be18-5fe4756f4649)
```
Comment (dustinbyrne) [inline description.md:11]: @jansorg Can we include images in the plugin description?

Comment (jansorg) [reply description.md:11]: yes, that's possible. But I recommend a static URL and not something GitHub could remove at any time. The marketplace could show the image even after the GitHub page changed.

Comment (Daniel-Warner-X) [reply description.md:11]: Image links updated.

Comment (Daniel-Warner-X) [reply description.md]: Image links updated @jansorg @dustinbyrne

**Review by dustinbyrne — APPROVED** [review]

**Pull request merged**